crates/kubelet/src/bootstrapping/mod.rs

```diff
@@ -23,10 +23,11 @@ const APPROVED_TYPE: &str = "Approved";
 pub async fn bootstrap<K: AsRef<Path>>(
     config: &KubeletConfig,
     bootstrap_file: K,
+    notify: impl Fn(String),
 ) -> anyhow::Result<Config> {
     debug!("Starting bootstrap for {}", config.node_name);
     let kubeconfig = bootstrap_auth(config, bootstrap_file).await?;
```

Comment:
Nit: Maybe we should have the same notify feature for auth in case there are distros that don't auto approve. That can probably be a low priority issue we create

Reply:
Deferring this after discussion with Taylor.